ci/build_container/build_container.sh

```diff
@@ -65,6 +65,16 @@ make install
 cd ..
 rm -fr nghttp2*
 
+# c-ares
+wget https://github.com/c-ares/c-ares/archive/cares-1_12_0.tar.gz
```

Comment:
Ah, yeah. I can go either way with this, I originally had a slight preference for using the tar ball connected with the public source and was trying to follow the other source fetches from github. Let me know which one is preferred.